snuba_sdk/expressions.py

```diff
@@ -155,7 +157,24 @@ class Debug(BooleanFlag):
 
 @dataclass(frozen=True)
 class Column(Expression):
+    """
+    A representation of a single column in the database. Columns are
+    expected to be alpha-numeric, with '.', '_`, and `:` allowed as well.
+    If the column is subscriptable then you can specify the column in the
+    form `column[key]`. The `column` attribute will contain the outer
+    column and `key` will contain the inner key.
+
+    :param name: The column name.
+    :type name: str
+
+    :raises InvalidExpression: If the column name is not a string or has an
+        invalid format.
+
+    """
+
     name: str
+    column: Optional[str] = field(init=False, default=None)
```

Comment:
The difference between column and name is kind of confusing. I would call column and keys something like `subscriptable` and `key`